metadata-models/src/main/pegasus/com/linkedin/datajob/DataJobInputOutput.pdl

```diff
@@ -60,4 +61,26 @@ record DataJobInputOutput {
     }
   }
   inputDatajobs: optional array[DataJobUrn] 
+
+  @Searchable = {
+    "/*": {
+      "fieldName": "inputFields",
+      "fieldType": "URN",
+      "numValuesFieldName": "numInputFields",
+      "queryByDefault": false
+    }
+  }
+  inputDatasetFields: optional array[DatasetFieldUrn]
```

Comment:
We try to avoid using Strongly Typed URNs anymore - please use generic Urn here!

Reply:
Fixed in latest commit, please check.